keycloak/themes/uid2-theme/login/template.ftl

```diff
@@ -13,7 +13,7 @@
         </#list>
     </#if>
     <title>${msg("loginTitle",(realm.displayName!''))}</title>
-    <link rel="icon" href="${url.resourcesPath}/img/favicon.ico" />
+    <link rel="icon" href="${url.resourcesPath}/img/keycloak_favicon.ico" />
```

Comment:
`keycloak_favicon.ico` is just a copy paste of `public/favicon.ico`. I haven't been able to figure out a way to just have one favicon that the keycloak page and the actual website both use. But open to ideas!

Reply:
fine for now